security/pfSense-pkg-snort/files/usr/local/www/snort/snort_interfaces_edit.php

```diff
@@ -243,6 +243,21 @@
 		}
 	}
 
+	if ($_POST['ips_mode'] == 'ips_mode_inline') {
+		$netmapifs = array('cxgbe', 'em', 'igb', 'em', 'lem', 'ixgbe', 'ixl', 're', 'vtnet');
```

Comment:
The `ixgbe` driver also covers `ix` interfaces.
The `cxgbe` driver also covers `cxl` and `cc`